src/tests/Directory.Build.targets

```diff
@@ -22,6 +22,20 @@
     <NoWarn Condition="'$(IsMergedTestRunnerAssembly)' == 'true'">$(NoWarn);CS2008</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <IlcCompileDependsOn Condition="'$(CustomIlcCompileDependsOn)' == 'true'">Compile;ComputeIlcCompileInputs;SetupOSSpecificProps;PrepareForILLink</IlcCompileDependsOn>
+  </PropertyGroup>
+
+  <Target Name="_SetCustomIlcPath" Condition="'$(CustomIlcCompileDependsOn)' == 'true'" BeforeTargets="SetupProperties">
```

Comment:
Why do we need to do this in a target? We already set IlcXXXPath in this file - can't we do it there?

Reply:
Apparently not needed, could be set in this file.